### PR TITLE
Update options for Picard 2.5

### PR DESCRIPTION
**Pull request by phw**
### Summary

This is a…

- [ ] Correction
- [x] Addition
- [ ] Restructuring
- [ ] Minor / simple change (like a typo)
- [ ] Other

### Reason for the Change

There have been several changes in the options for Picard 2.5:

- Support for SOCKS proxy
- Support for configuring a network timeout
- Cover art provider for approved URLs has been renamed for clarity (we probably could also remove it, but let's do this for a later time)
- Option to use system theme
- Built-in search is the new default

### CODE REVIEW

File: source/config/options_interface.rst [hunk 1]
```diff
@@ -18,6 +18,20 @@ User Interface Options
...
+    Picard will instead use the default Qt theme configured on your system. How to change the Qt
+    theme depends on your desktop environment.
+
+    .. note::
+        This option is not available on Windows and macOS.
+
+    .. warning::
+
+        Please be aware that using the system theme might cause the user interface to be not shown correctly.
+        If this is the case disable the "Use system theme" option to use Picard\'s default theme again.
```
Comment (rdswift) [inline source/config/options_interface.rst:33]: I don't think you need to escape the single quote.

File: source/config/options_network.rst [hunk 2]
```diff
@@ -15,8 +15,14 @@ Network
 
 **Web Proxy**
 
-    If you need a proxy to make an outside network connection you may specify one here.  The required
-    settings are **Server Address**, **Port**, **Username** and **Password**.
+    If you need a proxy to make an outside network connection you may specify one here.  You can
+    choose between HTTP and SOCKS proxy.  The required settings are **Server Address**, **Port**.
```
Comment (rdswift) [inline source/config/options_network.rst:19]: Perhaps "\*\*Server Address\*\* and \*\*Port\*\*"?

**Review by rdswift — APPROVED** [review]

**Comment by rdswift** [comment]
Also a note to myself to remind me that we will need to update the image file for the page, either after the v2.5 release or by running the source locally.

File: source/config/options_interface.rst (hunk 1 above)
Comment (phw) [reply source/config/options_interface.rst]: Yep, I copied this from Picard's source. Fixed

**Comment by phw** [comment]
Thanks for looking over this, fixed

**Pull request merged**